Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Core.cpp

```diff
@@ -287,6 +293,11 @@ namespace ScriptCanvas
 
     void SourceHandle::SanitizePaths()
     {
+        if (!m_relativePath.empty() && m_absolutePath.RelativePath() != m_relativePath)
+        {
+            m_absolutePath = m_relativePath;
```

Comment:
Question: do we still need abosolutePath? can we always use relative path instead?

Reply:
I'm investigating this.